### PR TITLE
Add project to the review branch

### PR DESCRIPTION
**Pull request by mrkamin**
review project from trail pull request

### CODE REVIEW

**Comment by mrkamin** [comment]
review of this pull by my self

File: README.md [hunk 1]
```diff
@@ -1,2 +1,2 @@
-# review-projects-from-trails
-this is for pull request review 
+# Review-of-past-projects
+This repo is made for reviewing of my Past Projects
```
Comment (mrkamin) [inline README.md:2]: no discriptive readme file

File: style.css [hunk 2]
```diff
@@ -0,0 +1,161 @@
...
+
+.submit-button {
+  display: block;
+  width: 100%;
+  padding: 0.75rem;
+  background: var(--color-green);
+  color: inherit;
+  border-radius: 2px;
+  cursor: pointer;
+}
```
Comment (mrkamin) [inline style.css:161]: Good job so far. Your website looks not completed I kindly suggest you code more to haveing a good website.
Capture

**Review by mrkamin — COMMENTED** [review]
There are some issues that you still need to work on to go to the next project, but you are almost there!
For more information, check the comments under the review ⬇️

File: style.css (hunk 2 above)
Comment (Nyame-Wolf) [reply style.css:161]: Good job so far. Your review however does not meet code review language requirements I suggest you make it clear, use polite language not commands and give a reason why you  requested a change:
This request is ambiguous. You should point out the specific part that needs more coding. An example of a clear comment:
**The README's "Built With" and "Getting Started" sections are currently empty. Why don't you add content, listing the technology used in the project and how to run it, to them so your README is more informative and professional?**

This is part of the project requirements.
Screenshot 2022-09-26 at 12 59 35

@Nyame-Wolf 
thanks for your good and valuable feedback.

**Review by Nyame-Wolf — CHANGES_REQUESTED** [review]
# STATUS: REQUIRED CHANGES :recycle: 

Hi @mrkamin  :wave: 

Your project page is looking fantastic so far :star_struck: ! 

However, there is still some room for improvement, that can help you have better code and make your project professional :sunglasses: 



## Highlights :sparkles:

- Great work using the right GitHub Flow :100: 
- Great work using the same font as Figma design ☑️ 
- Commits are descriptive and meaningful :+1: 
- Project Documentation (PR, README) is professional :necktie: 

Congrats on your work so far! :clap:  

## Required Changes ♻️

There are some issues that you still need to work on to go to the next project, but you are almost there! 
_For more information, check the comments under the review :arrow_down:_

## Optional suggestions :ballot_box_with_check: 

_Every comment with the **[OPTIONAL]** prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can improve the quality of your code :100:_

Happy coding and keep it up! :raised_hands: 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
  Please, remember to tag me **@Nyame-Wolf**  in your question so I can receive the notification.

  **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------

  _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)

File: README.md (hunk 1 above)
Comment (mrkamin) [reply README.md:2]: Your README file should be descriptive - all main headers from the [README template](https://github.com/microverseinc/readme-template/blob/master/README.md) should be present 👌 Kindly, update it 👌

**Review by frank1738 — APPROVED** [review]
Hi @mrkamin , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

YesYouAreGIF (2)

## Highlights

✅ Correct use of Github inline comments

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

@frank1738 
thanks for your good and valuable feedback.

File: style.css (hunk 2 above)
Comment (mrkamin) [reply style.css:161]: @Nyame-Wolf 
thanks for your good and valuable feedback.

**Pull request merged**